### PR TITLE
Add debuginfo flags for Fedora tests

### PR DESCRIPTION
**Pull request by hainest**

### CODE REVIEW

**Comment by vsoch** [comment]
@hainest this is going to run tests without the (not yet merged) incorrect add/diff branch of spliced, just FYI.

**Comment by hainest** [comment]
Yeah, I wasn't sure if I should wait on this. I figured we could manually cancel or just let it fail.

**Comment by vsoch** [comment]
Re-run just triggered - it should have the debug directories there now.

**Comment by vsoch** [comment]
@hainest it looks like it's broken - I think you are going to need to debug... sorry I'm not inside your head I'm not sure what you were trying to do, but ping me if you want to discuss anything!

**Comment by vsoch** [comment]
This took me 80+ times to get right, so... don't be discouraged? lol.

File: .github/workflows/test-fedora.yaml [hunk 1]
```diff
@@ -35,16 +35,27 @@ jobs:
 
       - name: Checkout
         uses: actions/checkout@v3
+      
+      - name: Install system debuginfo files
```
Comment (vsoch) [inline .github/workflows/test-fedora.yaml]: Niiice!

**Comment by hainest** [comment]
` name Remove` it's a missing colon. Derp!

**Comment by vsoch** [comment]
@hainest we are getting failures, I can only suspect the runner is out of space/memory. Is there another logical way we could break up the second set of runs?

**Comment by vsoch** [comment]
okay updated to try that - not sure it's a great idea if there is a dependency between two, but worth a shot. Ping me when you have time to chat about what other running strategies we can try.

**Comment by hainest** [comment]
If that doesn't fix it, then I would try again without installing the extra packages Ben suggested.

```diff
--- a/.github/workflows/test-fedora.yaml
+++ b/.github/workflows/test-fedora.yaml
@@ -44,7 +44,7 @@ jobs:
       - name: Install Fedora Libs
         run: |
           mkdir -p ./libs
-          /bin/bash fedora_installs.sh
+          #/bin/bash fedora_installs.sh
     
       - name: Install complete debuginfo files
         run: |
```

**Comment by vsoch** [comment]
What about deleting libs after we test them? I added coloring to your diff above so it's easier to read.

**Comment by vsoch** [comment]
okay so things to try:

 - [x] delete libs after we test [running here](https://github.com/buildsi/splice-experiment-runs/runs/8046109640?check_suite_focus=true) (failed)
 - [x] don't install extra libs [running here](https://github.com/buildsi/splice-experiment-runs/runs/8049779469?check_suite_focus=true) ran out of time (cancelled)
 - [x] don't create abi lab reports [running here](https://github.com/buildsi/splice-experiment-runs/runs/8047434664?check_suite_focus=true) ran out of time (not room) so could be tweaked?
 - [ ] try running with start and stops (0-5000) [running now](https://github.com/buildsi/splice-experiment-runs/actions/runs/2940990989)

**Review by vsoch — APPROVED** [review]

**Pull request merged**